DQ_Kinematics_test.py

```diff
@@ -16,43 +16,68 @@
     along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
 
 Contributors:
-- Murilo M. Marinho (murilomarinho@ieee.org)
+1. Murilo M. Marinho (murilomarinho@ieee.org)
+   - Responsible for the original implementation.
+2. Juan Jose Quiroz Omana (juanjqogm@gmail.com)
+   - Added tests for the DQ_SerialWholeBody class.
 """
 
 import unittest
 import scipy.io
 import numpy
-from dqrobotics.robots import KukaLw4Robot, KukaYoubotRobot
+from dqrobotics.robots import KukaLw4Robot, KukaYoubotRobot, FrankaEmikaPandaRobot
 from dqrobotics import *
 from DQ_test_facilities import *
+from dqrobotics.robot_modeling import DQ_SerialWholeBody
 
 # The data from MATLAB
 mat = scipy.io.loadmat('DQ_Kinematics_test.mat')
 # A list of random DQs
 h_list = get_list_of_dq_from_mat('random_dq_a', mat)
 # A list of random joint values
 serial_manipulator_q_list = get_list_of_vector_from_mat('random_q', mat)
+
+# A lists for the DQ_SerialWholeBody class
+serial_whole_body_q_list    = get_list_of_vector_from_mat('random_q_serial_whole_body', mat)
+serial_whole_body_pose_list = get_list_of_dq_from_mat('result_of_serial_whole_body_fkm', mat)
+serial_whole_body_raw_pose_list = get_list_of_dq_from_mat('result_of_serial_whole_body_raw_fkm', mat)
+serial_whole_body_pose_jacobian_list = get_list_of_matrices_from_mat('result_of_serial_whole_body_jacobian', mat)
```

Comment:
@juanjqo Is this being used?

Reply:
@mmmarinho no, I forgot to remove the parts related to the test of the `pose_jacobian` method. I removed it now.